tests/nn/pipe_process/__init__.py

```diff
@@ -1,21 +0,0 @@
-# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
-#
-# This source code is licensed under the BSD license found in the
-# LICENSE file in the root directory of this source tree.
-
-# Copyright 2019 Kakao Brain
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# tests/__init__.py makes pytest can import the application without custom sys.path or PYTHONPATH.
```

Comment:
this doesn't seem to apply to us. we don't have `__init__.py` in all test dirs and we import from fairscale.utils.testing for utils anyway. I think it is better to keep those file removed, esp. since they seem to confuse pytest and make pytest run additional tests not specified on the command line.